modules/op-release-notes-1-6.adoc

```diff
@@ -0,0 +1,194 @@
+
+* This update adds support for triggers resources to work with `v1beta1` resources (link:https://github.com/tektoncd/cli/pull/1446[#1446], link:https://github.com/tektoncd/cli/pull/1449[#1449], link:https://github.com/tektoncd/cli/pull/1450[#1450], link:https://github.com/tektoncd/cli/pull/1454[#1454], link:https://github.com/tektoncd/cli/pull/1455[#1455])
+
+* This update adds an `ignore-running` flag to the `tkn pipelinerun delete` and `tkn taskrun delete` commands. (link:https://github.com/tektoncd/cli/pull/1445[#1445])
+
+* This update adds a `create` subcommand to the `tkn task` and `tkn clustertask` commands. (link:https://github.com/tektoncd/cli/pull/1359[#1359])
+
+=== CLI 0.20.0 release
+
+* With this update, if you use the `tkn pipelinerun delete --all` command, you can preserve runs that are younger than a specified number of minutes by using the new `--keep-since <minutes>` option. For example, to keep pipeline runs that are less than five minutes old, you enter `$ tkn pipelinerun delete -all --keep-since 5`. (link:https://github.com/tektoncd/cli/pull/1392[#1392])
+
+* With this update, when you use the `tkn pipelinerun delete --all` command, you can use the new `--label <string>` option to filter the pipeline runs by label. Optionally, you can use the `--label` option with `=` and `==` as equality operators, or `!=` as an inequality operator. For example, the `tkn pipelinerun delete --all --label asdf` and  `tkn pipelinerun delete --all --label==asdf` commands both delete all the pipeline runs that have the `asdf` label. (link:https://github.com/tektoncd/cli/pull/1402[#1402])
+
+* With this update, you can fetch the version of installed Tekton components from the `ConfigMap` or, if the `ConfigMap` is not present, from the deployment controller. (link:https://github.com/tektoncd/cli/pull/1393[#1393])
+
+=== Tekton Triggers v0.16.0
+
+* With this update, triggers support the `configmap feature-flags` and `configmap config-defaults` commands to configure feature flags and to set default values, and adds `serviceAccountName` to the defaults. (link:https://github.com/tektoncd/cli/pull/1110[#1110], link:https://github.com/tektoncd/cli/pull/1182[#1182])
+
+* This update adds a new metric, `eventlistener_event_count`, that you can use to count events received by the `EventListener` sink. (link:https://github.com/tektoncd/triggers/pull/1160[#1160])
+
+=== Tekton Triggers v0.15.0
+
+* This update adds `v1beta1` Go API types. (link:https://github.com/tektoncd/triggers/pull/1103[#1103])
++
+With this update, Triggers now supports a new `apiVersion` value, `v1beta1`. The `v1alpha1` version will follow the beta deprecation policy: `v1alpha1` features will be deprecated over a 9-month period.
+
```

Comment:
In which section should I place the following release note? If you look at [the TOC in the preview](https://deploy-preview-37369--osdocs.netlify.app/openshift-enterprise/latest/cicd/pipelines/op-release-notes), I've placed it at the same level as:

- CLI 0.21.0 release
- CLI 0.20.0 release
- Tekton Triggers v0.16.0
- Tekton Triggers v0.15.0

However, it seems like I should place it _**within**_ one of those sections, or something similar. I couldn't determine this from the context. What do you think?

Reply:
Not sure I follow the question 🤔

Reply:
Oh, I think I understand. Configuring Node selection is a feature of the Operator (so OpenShift pipelines "installation")

Reply:
Thanks @vdemeester. I created the following `=== {pipelines-title} Operator` section and placed that item as a release note within it. How does that look to you?

Reply:
@vdemeester, I created the following `=== {pipelines-title} Operator` section to contain the release note. WDYT?